doc/ChangeLog

```diff
@@ -1,4 +1,231 @@
 ===============================================================
+Tag name: ctsm5.2.0
+Originator(s): erik (Erik Kluzek)
```

Comment:
@slevis-lmwg please review the suggested ChangeLog update for ctsm5.2.0.

An obvious thing I see now is that it should say multiple as the Originator...